### PR TITLE
Bug fix when receiveing msgs larger than 125 bytes.

### PR DESCRIPTION
**Pull request by palomovaldes**
Bug fix when receiveing msgs larger than 125 bytes. Msg was truncated cos of an error reading msg length.
Tested on windows.
Not tested with msgs even more larger

### CODE REVIEW

**Comment by silentbluedev** [comment]
Hi Palomovaldes, 
The problem is more of a TCP/IP / networking problem, the issues seems to be that your receiving buffer is low, (512 Bytes) and the clients is higher.

The client application / browser opens a socket and it's send buffer is 4096 bytes (default buffer size for sockets) the client application sends 600 bytes, this is ok for the client so data is sent.

The server TCP/IP stack receives 600 Bytes and then the server application gets 512 bytes on the socket because that's all you have allocated for the buffer, so the TCP/IP stack only gives you 512 Bytes the remaining 88 bytes if you don't process the 512 Bytes fast enough are dropped from the stack.

I was unable to find anything in the "draft-ietf-hybi-thewebsocketprotocol-17" about configuring or setting the socket buffer size in javascript so I would assume the default (4096 Bytes).

An interesting note from Windows Developer Preview is the "websocket.h" reference:
(http://msdn.microsoft.com/en-us/library/windows/desktop/hh449360(v=VS.85).aspx) 
Says the default microsoft uses the 4096 Bytes default and you can set. 

So to sum up... set your default buffer sizes to 4096 bytes.

Hope this helps...

p.s. Who can I talk to about adding to the project, as this project helped me work out protocal version handling. I have also implemented WSS support and would like to give that back to the project.

**Comment by palomovaldes** [comment]
thanks for your large mail.
I ll try it these days. 
Anyway I wounder what happens with even larger msgs. (I dont need it)

Want to join this project? I am new in github. You d better contact the owner at https://github.com/Hardt
I made a fork for a bug fix, not more by now.
i am learning about gihub and open source.

see u around

**Comment by silentbluedev** [comment]
Hi Palomovaldes,
Yeah sorry for the big email, but one of those things that is hard to explain correctly...

I am new to GitHub myself I have done a fork and just trying to connect from VS 2010

See you around too.

**Comment by palomovaldes** [comment]
Are you working on this?

I tryed with 4096 buffer. Now I receive the msg but its info is corrupted. Even the size is incorrect.
It seems to be you were write with buffer size (it crashed before), now we should take a look at the way we read the info from the buffer.

I think the problem is in hybi10\DataFrame.cs in Append method but it is not easy to see.
I should read carefully the websocket definition to learn the codification of received msg. Ufff....no time these days.

Welcome to github
Feel free to write, no problem with large mails!!!

c u!

**Comment by silentbluedev** [comment]
Hi, I am working on a project another project at the moment will get back to you asap.

**Comment by DorianGray** [comment]
This doesn't fix the issue. I'm gonna go ahead and close this pull request. Reopen it if you come up with something else. Otherwise I'll fix it when I get time.
